active_directory_ldap/plugin.spec.yaml

```diff
@@ -128,17 +141,19 @@ actions:
         description: Set this to true to disable the user account at creation
         required: true
         default: 'true'
+        example: 'true'
```

Comment:
```suggestion
        example: true
```

Reply:
removing these quotes cause a bug as the sdk interprets the example key as a Boolean which is now true. rather than example: value = true